frameworks/digital-outcomes-and-specialists/questions/brief-responses/newAvailabilityOutcomes.yml

```diff
@@ -0,0 +1,19 @@
+name: Earliest start date
+id: availability
+question: When is the earliest the team can start
```

Comment:
This appears to not be possible based on our current URL structure. We can merge the questions into one which is fine. However, then the section that they are used in also needs to be merged into one with jinja in the section name as such:

![image](https://cloud.githubusercontent.com/assets/7228605/20629196/1b7d24da-b322-11e6-8c0e-a399fa136e3d.png)

However, the problem with this is that the section name is used in the url and this will then 404 when matching by section name.

![image](https://cloud.githubusercontent.com/assets/7228605/20629210/3619250a-b322-11e6-999c-92ccbc00e198.png)